spec/Centipede/Filter/UrlFilterSpec.php

```diff
@@ -29,6 +24,6 @@ function it_filters_url_hash()
 
     function it_filters_trailing_slash()
     {
-        $this->filter('https://github.com/')->shouldReturn('https://github.com');
+        $this->filter('https://github.com/')->shouldReturn('https://github.com/');
```

Comment:
@Pierstoval Then we should remove this spec as it is invalid. Or it should say `it_des_not_filter_out_trailing_slash`?
Can you think of some other specs that we can add?